src/selectors/getChildren.ts

```diff
@@ -46,13 +47,20 @@ export const hasChildren = (state: State, context: Context) => {
 /** Gets all visible children within a context. */
 export const getChildren = getVisibleThoughts(getAllChildren)
 
-/** Gets all visible children within a context sorted by rank or sort preference. */
+/** Gets all children within a context sorted by rank or sort preference. */
 export const getChildrenSorted = (state: State, context: Context) => {
   const sortPreference = getSortPreference(state, context)
   const getThoughtsFunction = sortPreference === 'Alphabetical'
     ? getChildrenSortedAlphabetical
     : getChildrenRanked
-  return getVisibleThoughts(getThoughtsFunction, state, context)
+  return getThoughtsFunction(state, context)
+}
+
+/** Gets all visible children within a context sorted by rank or sort preference.
+ * Note: It doesn't check if thought lies within the cursor path.
+ */
+export const getVisibleChildrenSorted = (state: State, context: Context) => {
+  return getVisibleThoughts(getChildrenSorted, state, context)
 }
```

Comment:
Since `getVisibleThoughts` is curried, you can simplify as follows:

```js
export const getVisibleChildrenSorted = getVisibleThoughts(getChildrenSorted)
```